tools/int-to-hex.clj

```diff
@@ -0,0 +1,19 @@
+;; converts all int/long literals to hex
```

Comment:
Even if there was documentation for this one, I still wouldn't recommend committing it to the public repo. I don't think we want to expand the number of languages a user needs to have installed on their system to use the tooling. It could at least be written in Python to be consistent with the other scripts.

Reply:
I think it's fixed